pkg/api/configsync/v1beta1/gitconfig.go

```diff
@@ -25,11 +25,13 @@ type Git struct {
 	// repo is the git repository URL to sync from. Required.
 	Repo string `json:"repo"`
 
-	// branch is the git branch to checkout. Default: "master".
+	// branch is the git branch to checkout.
+	// The field is deprecated. Use `revision` instead.
+	// If both `branch` and `revision` are defined, `revision` takes precedence over `branch`.
 	// +optional
 	Branch string `json:"branch,omitempty"`
 
-	// revision is the git revision (tag, ref or commit) to fetch. Default: "HEAD".
+	// revision is the git revision (branch, tag, ref or commit) to fetch. Default: "HEAD".
```

Comment:
similar update should be made to the field on `v1alpha1`

Reply:
Good catch! done